x-pack/plugins/fleet/common/types/index.ts

```diff
@@ -29,6 +33,22 @@ export interface FleetConfigType {
     agentPolicyRolloutRateLimitIntervalMs: number;
     agentPolicyRolloutRateLimitRequestPerInterval: number;
   };
+  agentPolicies?: Array<
```

Comment:
now that we've deferred the kibana.yml requirement for now, we can remove changes to this file